### ISSUE TITLE
Unexpected compilation errors unknown:0:1:

### ISSUE THREAD

**Issue by VlachJosef** [Duhemm/sbt-errors-summary#23]
When trying to compile this simple program with one type error:
```scala
object Hello {
  final def main(args: Array[String]) = {
    val s: Int = "hello"
  }
}
```
Following are errors reported:
```
[error] [1] src/main/scala/main.scala:5:18:
[error]     type mismatch;
[error]      found   : String("hello")
[error]      required: Int
[error]         val s: Int = "hello"
[error]                      ^
[info] [2] unknown:0:1:
[info]     String("hello") <: Int?
[info]     
[info] [3] unknown:0:1:
[info]     false
[info]     
[error] src/main/scala/main.scala: 5 [1]
[error] unknown: 0 [2], 0 [3]
[error] (compile:compileIncremental) Compilation failed
```

Errors `[2]` and `[3]` should not be printed.

**Comment by Duhemm** [comment]
I can't reproduce it locally. Could you tell me what Scala version and compiler plugins you're using?

**Comment by VlachJosef** [comment]
Oh, I didn't realize I have `-explaintypes` in `scalacOptions`.

But still it may be little more convenient to print these information as sbt is doing:

```
[error] /Users/me/dev/sbt-errors-test/src/main/scala/main.scala:5: type mismatch;
[error]  found   : String("hello")
[error]  required: Int
[error]     val s: Int = "hello"
[error]                  ^
[info] String("hello") <: Int?
[info] false
[error] one error found
[error] (compile:compileIncremental) Compilation failed
```

**Comment by Duhemm** [comment]
Thanks! The way we show errors for "unknown" file must be reworked. 

Thanks for the report, il' see what I can do.

**Comment by Duhemm** [comment]
Fixed in #24
